operator_ui/src/containers/Notifications.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { receiveSignoutSuccess } from '../actions'
+import Flash from '../components/Flash'
+import Unhandled from '../components/Notifications/UnhandledError'
+
+const Error = ({ notifications }) => {
+  return (
+    <Flash error>
+      {notifications.map(({ component, props }, i) => {
+        if (component) {
+          return <p key={i}>{component(props)}</p>
+        } else if (props && props.msg) {
+          return <p key={i}>{props.msg}</p>
+        }
+
+        return (
+          <p key={i}>
+            <Unhandled />
+          </p>
+        )
+      })}
+    </Flash>
+  )
+}
+
+const Success = ({ notifications }) => {
+  return (
+    <Flash success>
+      {notifications.map(({ component, props }, i) => (
+        <p key={i}>{component(props)}</p>
+      ))}
+    </Flash>
+  )
+}
+
+interface IProps {
+  errors: any[]
+  successes: any[]
+}
+
+export const Notifications = ({ errors, successes }: IProps) => {
+  return (
+    <div>
+      {errors.length > 0 && <Error notifications={errors} />}
```

Comment:
implies we stack notifications right?
![image](https://user-images.githubusercontent.com/40662484/59819800-73ddbd80-92f7-11e9-9161-79d644499fd8.png)

I wonder if it's desired from ui POV